pkg/slayers/path/hopfield_spec.gobra

```diff
@@ -37,13 +37,12 @@ pure func ifsToIO_ifs(ifs uint16) option[io.IO_ifs]{
 ghost
 requires 0 <= start && start <= middle
 requires middle + HopLen <= end && end <= len(raw)
-requires acc(slices.AbsSlice_Bytes(raw, start, end), _)
+requires forall i int :: { &raw[i] } start <= i && i < end ==> acc(&raw[i], _)
 decreases
-pure func BytesToIO_HF(raw [] byte, start int, middle int, end int) (io.IO_HF) {
+pure func BytesToIO_HF_Helper(raw [] byte, start int, middle int, end int) (io.IO_HF) {
```

Comment:
These names are getting more and more unidiomatic :D I think calling these functions `BytesToHfSpec` and `BytesToHfSpecUnfolded` would be fine